### DISCUSSION TITLE
无法联网的服务端如何使用SWHL/table_rec_test_dataset

### DISCUSSION THREAD

**Discussion by SidneyRey** [SWHL/TableRecognitionMetric#2] [Q&A]
如题，服务器无法联网，怎么离线用这个数据集。
能否提供读取image 和html label的 datasets。感谢

**Comment by SWHL** [comment]
离线下载好就行了呀，就一个10.4M的压缩包。[link](https://huggingface.co/datasets/SWHL/table_rec_test_dataset/tree/main/data)
